python/tvm/topi/cuda/scatter.py

```diff
@@ -764,6 +764,9 @@ def scatter_nd(data, indices, updates, mode):
     """
     _verify_scatter_nd_inputs(data, indices, updates)
 
+    def cur_target_kind(kind="cuda"):
+        return tvm.target.Target.current(allow_none=False).kind == tvm.target.Target(kind).kind
```

Comment:
`tvm.target.Target.current(allow_none=False).kind == kind`

Reply:
Fixed

I use `str(tvm.target.Target.current(allow_none=False).kind)`
convert `<class 'tvm.target.target.TargetKind'>` to  `string`